### PR TITLE
Creses and SuiteSparse fix

### PR DESCRIPTION
**Pull request by KindDragon**
Part of changes from #1390

### CODE REVIEW

**Comment by ras0219-msft** [comment]
Thanks!

File: ports/ceres/portfile.cmake [hunk 1]
```diff
@@ -55,17 +55,20 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
-
 vcpkg_fixup_cmake_targets(CONFIG_PATH "CMake")
 
+file(READ ${CURRENT_PACKAGES_DIR}/share/${PORT}/CeresConfig.cmake CERES_MODULE)
+string(REPLACE "\${CERES_CURRENT_CONFIG_DIR}/../" "\${CERES_CURRENT_CONFIG_DIR}/../../" CERES_MODULE "${CERES_MODULE}")
```
Comment (KindDragon) [inline ports/ceres/portfile.cmake]: Why is this necessary?
My CeresConfig contains this without you changes:
```
get_filename_component(CURRENT_ROOT_INSTALL_DIR
    ${CERES_CURRENT_CONFIG_DIR}/../../
    ABSOLUTE)
```
